pkg/client-sdk/client/client.go

```diff
@@ -43,6 +45,12 @@ type ASPClient interface {
 	CompletePayment(
 		ctx context.Context, signedRedeemTx string, signedUnconditionalForfeitTxs []string,
 	) error
+	SendTreeNonces(
+		ctx context.Context, roundID, cosignerPubkey string, nonces bitcointree.TreeNonces,
+	) error
+	SendTreeSignatures(
+		ctx context.Context, roundID, cosignerPubkey string, signatures bitcointree.TreePartialSigs,
+	) error
 	Close()
```

Comment:
Can we do bundle this in a single RPC?

Reply:
No, musig2 **must** be 2 rounds as far as I get it